packages/react-native-bridge/android/react-native-bridge/build.gradle

```diff
@@ -96,7 +96,8 @@ dependencies {
         implementation "com.github.wordpress-mobile:react-native-reanimated:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-reanimated', 'dependencies')}"
         implementation "com.github.wordpress-mobile:react-native-prompt-android:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-prompt-android', 'dependencies')}"
         implementation "com.github.wordpress-mobile:react-native-webview:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-webview', 'dependencies')}"
-        runtimeOnly "org.wordpress-mobile:hermes-release-mirror:0.64.0"
+        implementation "com.github.wordpress-mobile:react-native-clipboard:${extractPackageVersion('../../../react-native-editor/package.json', '@react-native-clipboard/clipboard', 'dependencies')}"
+        runtimeOnly "org.wordpress-mobile:hermes-release-mirror:0.66.1"
```

Comment:
I understand we're targeting the `.2` of `0.66.2` currently, should we also update it here?

Reply:
Good catch. I addressed this in https://github.com/WordPress/gutenberg/pull/36328/commits/c3307b48de08df88851d9d16f09d764c350ce1e3.